cardigan/stories/global/palette/palette.stories.tsx

```diff
@@ -56,9 +56,10 @@ const PaletteColor = styled.div<{ hasBorder: boolean }>`
   margin-right: 15px;
 
   /* stylelint-disable value-keyword-case */
-  border: 1px solid
-    ${props =>
-      props.hasBorder ? props.theme.color('neutral.500') : 'transparent'};
+  border: ${props =>
+    `1px solid ${
+      props.hasBorder ? props.theme.color('neutral.500') : 'transparent'
+    }`};
```

Comment:
had to do this otherwise there was extra spacing and the linter was unhappy. this was caused by the new `declaration-property-value-no-unknown` rule and was the only instance of extra spacing it complained about.